.gitignore

```diff
@@ -4,3 +4,4 @@ public
 node_modules
 .nyc_output
 dump.rdb
+config/
```

Comment:
Can you remove this please as I'll be adding a config directory in a few minutes.